compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -476,7 +476,7 @@ message FailedAuthorization {
 }
 
 message FailedAuthorizations {
-  repeated FailedAuthorization failed_authorizations = 1;
+  repeated FailedAuthorization failed_authorizations = 1; //at most one now
```

Comment:
Why not dropping completely the `repeated` ?
We are not maintaining any backward compatibility for this proto.

Reply:
ok, I can make these changes here.
I guess this will need changes on the haskell side too.

Reply:
if this implies too much change in the haskell side, you can let it like that.

Reply:
It's probably not too much. But I will leave it for a future PR

Reply:
fine with me.